src/main/java/org/jenkinsci/lib/configprovider/ConfigProvider.java

```diff
@@ -134,8 +134,10 @@ public Config newConfig(@NonNull String id, String name, String comment, String
 
     private void setField(String fieldName, String value, Config config) {
         Field field = ReflectionUtils.findField(config.getClass(), fieldName);
-        field.setAccessible(true);
-        ReflectionUtils.setField(field, config, value);
+        if (field != null) {
```

Comment:
if the field is `null` won't this cause failures later on?
presumably this was trying to set the field to actually accomplish something so failing silently would seem bad.

Reply:
The fields set are only `name`, `comment`, `content` which are in the base `Config` class. So there is no way to not find them.